I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 216, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 531, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 531, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 542, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
AttributeError: Can't get attribute 'quest' on <renpy.python.StoreModule object at 0x105402190>

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 216, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 531, in script
    python hide:
  File "/Applications/Ren'Py/renpy/ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/Ren'Py/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00keymap.rpy", line 531, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 542, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
  File "/Applications/Ren'Py/renpy/loadsave.py", line 828, in load
    roots, log = loads(log_data)
  File "/Applications/Ren'Py/renpy/compat/pickle.py", line 100, in loads
    return load(io.BytesIO(s))
  File "/Applications/Ren'Py/renpy/compat/pickle.py", line 97, in load
    return up.load()
  File "/Applications/Ren'Py/renpy/compat/pickle.py", line 93, in find_class
    return super().find_class(module, name)
AttributeError: Can't get attribute 'quest' on <renpy.python.StoreModule object at 0x105402190>

macOS-14.5-arm64-arm-64bit arm64
Ren'Py 8.2.3.24061702
Office Simulator 2025 InfDev
Tue Aug  6 17:51:01 2024
